### ISSUE TITLE
Rendering of scenery shows visual artifacts

### ISSUE THREAD

**Issue by osconnoisseur** [OpenRCT2/OpenRCT2#2334]
These are two screenshots of the same park at the same zoom level. There are slight differences between the two, most notably the corrugated roofs and the gray silo at the bottom of the picture.

Original RCT2:
![scr6](https://cloud.githubusercontent.com/assets/15527244/11230975/c21dbc6c-8d69-11e5-8e78-f43171333494.png)

OpenRCT2
![scr2](https://cloud.githubusercontent.com/assets/15527244/11230959/a40543ee-8d69-11e5-87c2-2c3b40ea16ec.png)

**Comment by osconnoisseur** [comment]
You may have to zoom in a bit to see these images more clearly. The pixel pattern on the flat red building is different, and the small, gray tank/silo has a white vertical line running through it

**Comment by duncanspumpkin** [comment]
Yeah its never been completely perfect the new code but its a real pain to adjust.

**Comment by Gymnasiast** [comment]
As of https://github.com/OpenRCT2/OpenRCT2/commit/c0eade9c88ec95882a20bf1959b5c8d69248371b , the flat brown building looks correct. The gray silo looks correct in software rendering mode, but not in OpenGL mode.
